python/cudf/cudf/core/column/timedelta.py

```diff
@@ -149,9 +150,16 @@ def to_pandas(
         # Workaround until following issue is fixed:
         # https://issues.apache.org/jira/browse/ARROW-9772
 
-        # Pandas supports only `timedelta64[ns]`, hence the cast.
+        if PANDAS_GE_200:
+            host_values = self.fillna("NaT").values_host
```

Comment:
Does cudf support day or minute resolutions? Those would probably need to be downcast to seconds if so

Reply:
cudf doesn't support those time resolutions. However, we do downcast to `seconds` if we receive those inputs here: https://github.com/rapidsai/cudf/blob/branch-23.06/python/cudf/cudf/core/column/column.py#L2077